src/app/pages/send/send-crypto-asset-form/components/amount-field.tsx

```diff
@@ -36,23 +36,29 @@ interface AmountFieldProps {
   bottomInputOverlay?: JSX.Element;
   autofocus?: boolean;
   autoComplete?: 'on' | 'off';
+  tokenSymbol?: string;
 }
 export function AmountField({
   balance,
   switchableAmount,
   bottomInputOverlay,
   autofocus = false,
   autoComplete = 'on',
+  tokenSymbol,
 }: AmountFieldProps) {
   const [field, meta] = useField('amount');
   const [fontSize, setFontSize] = useState(maxFontSize);
   const [previousTextLength, setPreviousTextLength] = useState(1);
 
-  const { decimals, symbol } = balance;
+  const { decimals } = balance;
+  const symbol = tokenSymbol || balance.symbol;
   const maxLength = decimals === 0 ? maxLengthDefault : decimals + 2;
   const fontSizeModifier = (maxFontSize - minFontSize) / maxLength;
 
   useEffect(() => {
+    if (symbol.length > 4) {
+      setFontSize(24);
```

Comment:
Might read better if this was done on one line, but not critical. Curious the significance of `24`, should that be set as a named constant?

Reply:
yeah, I used `minFontSize` constant